Sources/RxAlamofire.swift

```diff
@@ -784,6 +790,25 @@ extension Reactive where Base: DataRequest {
         }
     }
 
+    public func responseJSON() -> Observable<DataResponse<Any>> {
+        return Observable.create { observer in
+            let request = self.base
+
+            request.responseJSON { response in
+                if let error = response.result.error {
+                    observer.onError(error)
+                } else {
+                    observer.onNext(response)
+                    observer.onCompleted()
+                }
+            }
+
+            return Disposables.create {
```

Comment:
Actually this would have to be 
`Disposables.create(with: request.cancel)` 

Which really isn't as neat. I think I'll leave this with a regular closure as its a bit easier on the eye.